client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
```

Comment:
Sorry, i didn't explain that correctly. It breaks if I changes lines 170-174 and 179 to ```this.elem``` as well